controllers/destination.controller.ts

```diff
@@ -0,0 +1,24 @@
+import { Context } from "koa";
+import { ProcessorRequest } from "../types/types";
+import { WorkFlowSelectorService } from "../services/worflowSelector.service";
+import IWorkFlow from "../services/IWorkFlow";
+
+export class DestinationController {
+  public static async destinationVanillaTransformAtProcessor(ctx: Context) {
+    const events = ctx.request.body as ProcessorRequest[];
+    const { version, destination } = ctx.params;
+    const reqParams = ctx.request.query;
+    const destinationHandler = WorkFlowSelectorService.getDestHandler(
```

Comment:
destination handler and workflow service both needed, is it for comparison logic?

Reply:
I am passing the handler as an argument for the workflow service. Handler is the core `transform.js` functions which I am decoupling from the workflow logic this way I will be able to handle other intermediate services (comparator or other stuff if required)